CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributions
+
+For contributions please:
+1. Make sure your fork (fork it if it's the first time) is in sync with the parent repository.
+2. Make your changes in a topic branch in your fork.
+2. Make sure the changes pass all the tests & do not break anything.
+3. When ready create a pull request from your fork to the parent repository.
+
+# Local Setup
+
+## Initial Setup
+
+### Pre-requisites
+
+1. Install Rust: https://www.rust-lang.org/tools/install
+2. Install Docker: https://docs.docker.com/engine/install
+
+Install Make:
+
+```shell
+sudo apt-get install build-essential
+```
+
+Install `parallelrun` to run commands in parallel:
+
+```shell
+cargo install parallelrun
```

Comment:
or `cargo binstall parallelrun` if the user uses that

Reply:
👍  addressed in https://github.com/liveask/liveask/pull/81/commits/2ac54bedb64638d3390d5e1f004a90ffb7a7fbb8